File: gamsojji/backjoon/step8/1193.js
let fs = require("fs");
const filePath = process.platform === 'linux' ? "/dev/stdin" : `${__dirname}/input.txt`;
let num = fs.readFileSync(filePath).toString().trim();
let bunza = 0, bummo = 0;
let t = Number(num);
let n = 2;
while(t > n){
    t -= n-1;
    n++;
}
if(n%2 == 0){
    bunza = n-t;
    bunmo = t
}else{
    bunza = t
    bunmo = n - t
}
console.log(bunza+"/"+bunmo);

File: rolemadelen/baekjoon/step8/1193.rs
use std::io::{stdin};

fn main() {
    let mut input = String::new();
    stdin().read_line(&mut input).unwrap();
    let mut n: i32 = input.trim().parse().unwrap();

    let mut i = 1;
    while n > i {
        n -= i;
        i += 1;
    }

    if i % 2 == 1 {
        println!("{}/{}", i+1-n, n);
    } else {
        println!("{}/{}", n,i+1-n);
    }
}
